tests/IntegrationTests/DeadLetters.fs

```diff
@@ -0,0 +1,215 @@
+module Pulsar.Client.IntegrationTests.DeadLetters
+
+open System
+open Expecto
+open Pulsar.Client.Api
+open FSharp.Control.Tasks.V2.ContextInsensitive
+open System.Threading.Tasks
+open Pulsar.Client.Common
+open Serilog
+open Pulsar.Client.IntegrationTests.Common
+
+[<Tests>]
+let tests =
+
+    let client = getClient()
+
+    let newGuid() = Guid.NewGuid().ToString("N")
+    let getTopicName() = "public/default/topic-" + newGuid()
+    let getProducerName() = sprintf "dlqProducer-%s" (newGuid())
+    let getConsumerName() = sprintf "negativeConsumer-%s" (newGuid())
+    let getDlqConsumerName() = sprintf "dlqConsumer-%s" (newGuid())
+    let getDeadLettersPolicy() = DeadLettersPolicy(0, sprintf "public/default/topic-%s-DLQ" (newGuid()))
+
+    let subscriptionName = "dlqSubscription"
+    let numberOfMessages = 50
+
+    let logTestStart testDescription = Log.Debug(sprintf "Started '%s'" testDescription)
+    let logTestEnd testDescription = Log.Debug(sprintf "Finished '%s'" testDescription)
+
+    let buildProducer(producerName, topicName) =
+        ProducerBuilder(client)
+            .ProducerName(producerName)
+            .Topic(topicName)
+            .EnableBatching(false)
```

Comment:
I suggest reverting logic by removing this statemtn, since EnableBatching is true by default. To avoid chain .EnableBatching(false).EnableBatching(true)

Reply:
Done.